pages/services/mongodb-enterprise/0.1.0-4.0.3/quick-start-guide/index.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: layout.pug
+navigationTitle: Quick Start
+excerpt: Configuring MongoDB - Quick Start
+title: Quick Start
+menuWeight: 15
+---
+
+# How to use MongoDB with DC/OS
+
+## Prerequisites
+
+* A running DC/OS cluster
+
+* OpsManager Base URI (Host URL or Path where the OpsManager is hosted)
+
+* Project ID (Obtained after creating a project on OpsManager UI)
+
+* Generate new or use an existing Public API Key from OpsManager.
+
+* Username of the Registered user over OpsManager Organisation.
+
+* Whitelist the IP of host in the OpsManager.
+
+
+## Install MongoDB OpsManager
+
+For Installing MongoDB OpsManager on a host machine, please gather the requirements specified and follow the steps provided on the [MongoDB OpsManager Installation](https://docs.opsmanager.mongodb.com/current/installation/) page.
+
+
+## Install MongoDB Service on DC/OS
+
+MongoDB can be installed via either the DC/OS Catalog web interface or by using the CLI. 
+
+Below are the steps to install MongoDB using the DC/OS Catalog Web Interface:
+
```

Comment:
There are no steps, no text, only a few screencaps. Please add steps explaining what users see in the screen grabs and what actions they need to take.

Reply:
Hi @SarahStegall
Required steps and text are added.
cc : @ryadav88